common/djangoapps/microsite_configuration/backends/base.py

```diff
@@ -272,9 +271,9 @@ def enable_microsites(self, log):
         Configure the paths for the microsites feature
         """
         microsites_root = settings.MICROSITE_ROOT_DIR
-
+        from edxmako.paths import add_lookup
```

Comment:
I can't remember -- are we doing this function-level import instead of module-level because of a circular reference issue?  It seems like an odd place to have the import especially because it's not protected by any sort of guard clause.